### PR TITLE
Sets a maximum and minimum length for password field in Sign Up form. Fix issue #578

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Set min and max length for password in sign up form

Fixes # 578

# Type of Change:
- Code
- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] Any dependent changes have been merged 
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

@tapasweni-pathak @tonythomas01 Please review this

### CODE REVIEW

**Comment by rohan033** [comment]
@sammy1997 Please also update the corresponding template to warn users about the password length.

File: docs/Installation Guide.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If you don't already have git, you can download it [here](http://git-scm.com/dow
 
 Clone the project from GitHub by running the following command:
 
-    git clone project_url_here
+    git clone https://github.com/systers/vms
```
Comment (tonythomas01) [inline docs/Installation Guide.md]: I think this is unnecessary, as check the next line 

```
 For this project, it would correspond to:

```

File: vms/registration/forms.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 class UserForm(forms.ModelForm):
     # password not visible when user types it out
-    password = forms.CharField(widget=forms.PasswordInput())
+    password = forms.CharField(widget=forms.PasswordInput(), min_length=8, max_length=25)
```
Comment (tonythomas01) [inline vms/registration/forms.py]: Why do we have two unrelated changes in one PR ?

File: docs/Installation Guide.md (hunk 1 above)
Comment (sammy1997) [reply docs/Installation Guide.md]: This change wasn't made by me. @tonythomas01 This was pushed by @Dr-G and @willingc . This was added as I tried to merge the changes. The main change was in commit https://github.com/systers/vms/pull/590/commits/2bf3a41f58ca92da176920be8332735547761ab3.

File: vms/registration/forms.py (hunk 2 above)
Comment (sammy1997) [reply vms/registration/forms.py]: This is the change which fixes issue #578 . The other one is a merged commit. @tonythomas01

File: docs/Installation Guide.md (hunk 1 above)
Comment (sammy1997) [reply docs/Installation Guide.md]: @tonythomas01 What should I do in this case?

Comment (sammy1997) [reply docs/Installation Guide.md]: I also cherry-picked my commit. But these merged commits still appeared. Thanks.

Comment (anjali-dhanuka) [reply docs/Installation Guide.md]: @sammy1997 I guess you must rebase your commits.

Comment (sammy1997) [reply docs/Installation Guide.md]: @anjali-dhanuka Thanks. I completely forgot to rebase my branch.

Comment (sammy1997) [reply docs/Installation Guide.md]: @tonythomas01 I will do this and send a new PR.

**Comment by sammy1997** [comment]
@rohan033 That's already there. This generates a form error. Which is already accounted for in the templates.
